Classes/Configuration/PhinxConfiguration.php

```diff
@@ -31,7 +31,7 @@ public function toArray(): array
                 'typo3' => [
                     'adapter' => 'mysql',
                     'host' => $connectionParameters['host'],
-                    'port' => $connectionParameters['port'],
+                    'port' => $connectionParameters['port'] ?? 3306,
```

Comment:
What about changing the default to `null`, as the database library (e.g. Doctrine) already provides a fallback. I can confirm, that it is working with `null` as well.

Reply:
Sounds reasonable and would make the config a bit more generic to support other `adapter`s in the future.